pkgs/development/python-modules/linkify-it-py/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, coverage
+, pytest-cov
+, uc-micro-py
+}:
+
+buildPythonPackage rec {
+  pname = "linkify-it-py";
+  version = "1.0.1";
```

Only [Python > 3.6](https://github.com/tsutsu3/linkify-it-py/blob/main/setup.py#L37) is  supported.

```suggestion
  version = "1.0.1";
  disabled = pythonOlder "3.6";
```

---

Given that we dropped Python < 3.6 a while ago, does it make sense to use that instead of e.g. `isPy27`? (I don't pay attention to `python2Packages` so I don't know what's current practice.)

---

I think this is more aligned with what upstream intended. And it's a better experience for people to get the disabled error, than a failing build